### ISSUE TITLE
Copyright

### ISSUE THREAD

**Issue by balping** [BraziDev/ticketit#1]
Hi @brazitech 

You've got no right to change the copyright information as you did it in this particular commit: 

https://github.com/BraziDev/ticketit/blob/e659cb1dc9af45c7ba40dac557ffee023c25a0b5/LICENSE.txt#L3

from _Ahmed M. Kordy_ to _Ahmed M. Brazidev_ . Unless you're the same person as @thekordy but then why use two names?

So please either delete this repo or change the copyright info back.

**Comment by balping** [comment]
From the MIT license:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

What you're doing is a violation of this paragraph.

You can however hold copyright for the changes you made, but not for the whole package. The MIT license has very few limitations, this is one of them.

**Comment by thekordy** [comment]
No, this is not me. Please @brazitech change the copyright info back, as @balping wrote, you don't have the right to change the copyright.
